source/renderer/app/api/api.js

```diff
@@ -1272,6 +1274,24 @@ export default class AdaApi {
     }
   };
 
+  getICOPublicKey = async (
+    request: ICOPublicKeyParams
+  ): Promise<string> | null => {
```

Comment:
Do we need `null` here?

Reply:
What do you like more? 'any' or 'string | null' ?